duckduckgo/Android
# Feature/investigate tablet support

Asana Issue URL: https://app.asana.com/0/414730916066338/530655794274203

## Description
Improves the onboarding screens when viewed on tablets


## Steps to Test this PR:
1. View onboarding on a range of tablets in both portrait and landscape
1. Ensure experience on phones still look good
